sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -188,21 +202,24 @@ def __repr__(self):
 class FormField(object):
     """Represents a field recognized in an input form.
 
+    :ivar type: The type of the value found on FormField. Possible types include: 'string',
+        'date', 'time', 'phoneNumber', 'number', 'integer', 'object', or 'array'.
+    :vartype type: str or ~azure.ai.formrecognizer.FieldValueType
     :ivar ~azure.ai.formrecognizer.FieldData label_data:
         Contains the text, bounding box, and field elements for the field label.
     :ivar ~azure.ai.formrecognizer.FieldData value_data:
         Contains the text, bounding box, and field elements for the field value.
     :ivar str name: The unique name of the field or label.
     :ivar value:
-        The value for the recognized field. Possible types include: 'string',
-        'date', 'time', 'phoneNumber', 'number', 'integer', 'object', or 'array'.
+        The value for the recognized field.
```

Comment:
nit: I would somehow link this to the variable `type`. Something along the lines of
"The value for the recognized field. It's data type is described in `type`"

Reply:
added!